modules/ting_fulltext/js/ting_fulltext.js

```diff
@@ -0,0 +1,14 @@
+(function ($) {
+    "use strict";
+
+    var SearchExpansionResults = function (response) {
+        $("#ting-lektor-fulltext").html(response);
+    }
+
+    Drupal.behaviors.fullTextReview = {
+        attach: function (context) {
+            $.get(Drupal.settings.tingFulltext.url, null, SearchExpansionResults);
+        }
+    };
```

Comment:
Drupal behaviors can trigger multiple times on a page e.g. for the initial pageload and if new content is loaded via AJAX. The current implementation will cause the fulltext content to load multiple times. Also the current implementation requires only a single instance of the fulltext element. This maybe correct at the moment but it is not something that we know to hold.

I suggest that the current approach is updated to look for fulltext containers (probably identified with a `js-ting-fulltext-lektor` class or the like) within `context` and that the data needed is retrieved from a url set as a data-attribute within that element instead of a global JavaScript setting.